### PR TITLE
Added controls for Table Viz

### PR DESCRIPTION
**Pull request by vera-liu**
Done:
- Added NotGroupBy and Options in control for Table Viz
  screen shot 2016-10-04 at 4 24 22 pm

@ascott 
Todo: 
- tests

### CODE REVIEW

File: caravel/assets/spec/javascripts/explore/components/actions_spec.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ describe('reducers', () => {
   });
 
   it('should return new state with time stamp', () => {
-    const newState = exploreReducer(initialState, actions.setTimeStamp(1));
+    const newState = exploreReducer(initialState, actions.setTimeStampFormat(1));
```
Comment (vera-liu) [inline caravel/assets/spec/javascripts/explore/components/actions_spec.js]: Change function name here to match with the variable name in store

File: caravel/assets/javascripts/explorev2/constants.js [hunk 2]
```diff
@@ -33,3 +42,35 @@ export const sinceOptions = ['1 hour ago', '12 hours ago', '1 day ago',
...
+);
+
+export const TableVizControls = (
+  <div>
+    <NotGroupBy />
+    <Options />
+  </div>
+);
+
+export const VIZ_CONTROL_MAPPING = {
```
Comment (vera-liu) [inline caravel/assets/javascripts/explorev2/constants.js]: The viz-control panel mapping is stored here

File: caravel/assets/javascripts/explorev2/components/ControlPanelsContainer.jsx [hunk 3]
```diff
@@ -1,20 +1,38 @@
...
+const propTypes = {
+  vizType: React.PropTypes.string,
 };
-export default ControlPanelsContainer;
+
+const defaultProps = {
+  vizType: null,
+};
+
+class ControlPanelsContainer extends React.Component {
```
Comment (vera-liu) [inline caravel/assets/javascripts/explorev2/components/ControlPanelsContainer.jsx]: Getting linting error here saying component should be written as a pure function, but in stateless components 'this.props' can't be accessed

Comment (williaster) [inline caravel/assets/javascripts/explorev2/components/ControlPanelsContainer.jsx]: typically if you write a pure function you do so like:
`function ControlPanelsContainer(props) { ... }`

and so you access `props` the same way you would acces `this.props` if it were a class.

**Comment by ascott** [comment]
looks good! :stamp:

**Comment by ascott** [comment]
looks like you need to rebase master to fix the conflicts in views.py

**Comment by vera-liu** [comment]
Solved and merged!

**Pull request merged**